tests/ut/ut_scargo_build.py

```diff
@@ -1,9 +1,11 @@
 from pathlib import Path
 
+from pytest_subprocess import FakeProcess
+
 from scargo.scargo_src.sc_build import scargo_build
 
 
-def test_scargo_build_dir_exist(fp, create_new_project):
+def test_scargo_build_dir_exist(fp: FakeProcess, create_new_project: None) -> None:
```

Comment:
Seems really weird to have return type for tests, don't really know what to think

Reply:
It is a little weird, but it's easier to just add them than to make elaborate ignore rules.

Reply:
Sure, if there is no simple way to do it.